mypy/main.py

```diff
@@ -527,6 +527,13 @@ def add_invertible_flag(flag: str,
                         help="Disallow decorating typed functions with untyped decorators",
                         group=untyped_group)
 
+    toplevel_group = parser.add_argument_group(
```

Comment:
Shouldn't this be added to `strictness_group` rather than making a new toplevel group?